packages/seedless-onboarding-controller/package.json

```diff
@@ -50,7 +50,7 @@
     "@metamask/auth-network-utils": "^0.3.0",
     "@metamask/base-controller": "^9.0.0",
     "@metamask/messenger": "^0.3.0",
-    "@metamask/toprf-secure-backup": "^0.7.1",
+    "@metamask/toprf-secure-backup": "^0.9.0",
```

Comment:
one more version of toprf is going to be released today, lets merge that along with this PR